exotic/exotic.py

```diff
@@ -2231,6 +2231,12 @@ def main():
 
             exotic_infoDict['exposure'] = exp_time_med(exptimes)
 
+            # save PSF data to disk using savetxt
+            np.savetxt(Path(exotic_infoDict['save']) / "temp" / "psf_data_target.txt", psf_data["target"],
+                          header="#x_centroid, y_centroid, amplitude, sigma_x, sigma_y, rotation offset",
+                          fmt="%.6f")
+                        # x-cent, y-cent, amplitude, sigma-x, sigma-y, rotation, offset
+
             # PSF flux
             tFlux = 2 * np.pi * psf_data['target'][:, 2] * psf_data['target'][:, 3] * psf_data['target'][:, 4]
```

Comment:
Look through the file for ints in float operations that may result in inadvertent rounding. Explicit is better than implicit.

Reply:
All of those numbers should be floats

Reply:
Python defaults to float with int * float. Explicit helps with inference/readability for some casual viewers.